wiki/introduction/get-started/en.md

```diff
@@ -32,19 +42,23 @@ Asking questions properly, whether it is on our forum or on Discord, is an essen
 skill to have. Never be afraid to ask questions, we're always willing to help, and we
 don't judge.
 
-TODO: Methodology for asking questions here
+When asking questions, try to include the necessary context in your first message, and [do not ask to ask](https://dontasktoask.com/). State your problem, what you have already tried to solve it, and how those attempts failed. Make sure to avoid the [XY problem](https://xyproblem.info/) by not only asking for help for one specific part of a solution to your problem, but also stating the actual problem you are trying to solve. When debugging, include your `latest.log` and when your game crashes, a crash log. Additionally, full access to your source code can be really helpful, ideally uploaded to a version control system such as GitHub.
+
+You can ask development related questions in our [Forum](https://forum.quiltmc.org/), as well as in the [mod-dev-help](https://discord.com/channels/817576132726620200/1047429688521396325) forum channel in our [Discord](https://discord.quiltmc.org/)
 
 Many problems have been solved in modding before, and are publicly available for you
 to see and use as most mods for Quilt are open source. We recommend that you look at
 examples of how to do things in open source repositories and sometimes "steal" code.
 Open source repositories are there for you to learn from and use, don't be afraid of
 them!
 
-TODO: Levi write your thing about stealing code here
+There are many topics in modding where you quickly will find yourself at a point where tutorials are missing. In those cases you'll usually either have to understand the relevant Minecraft code, or look at other mods and see how they got similar things done.
```

Comment:
```suggestion
There are many topics in modding where you quickly will find yourself at a point where there are no tutorials to help. In those cases you'll usually either have to understand the relevant Minecraft code, or look at other mods and see how they got similar things done.
```
I think a quick explainer on how to see game code (vineflower gen sources) would fit nicely here